### PR TITLE
added testing documentation to both the doc folder and testing folder

### PR DESCRIPTION
**Pull request by clmedders**
**_Delete all details that do not apply to this PR!_**

## Jira ticket (required)
[Vote-1338](https://bixal-projects.atlassian.net/jira/software/c/projects/VOTE/boards/254?modal=detail&selectedIssue=VOTE-1338&assignee=611272e0eb7aef00694b8a97)

## Description (optional)
This PR will add testing documentation to the project 

### Testing steps
1. Review files changed and ensure the information that is presented is relevant and current
2. click though links and ensure they are taking you to the correct place

### CODE REVIEW

**Review by nehemiah-abuga — DISMISSED** [review]
Looks good. Approved

File: testing/docs/gettingStarted.md
Comment (rayestrada) [inline testing/docs/gettingStarted.md:1]: Change the name of this file to README.md and move to `testing/README.md`

File: testing/docs/gettingStarted.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+_Before running these steps please visit the [testing.md](../../docs/testing.md) for a better understand of the overall testing coverage and approach for the project_
+
+__Please note all scripts/commands must be run in the `testing` folder__
+
+See [Installing Cypress Documentation](https://docs.cypress.io/guides/getting-started/installing-cypress#System-requirements) to get Cypress set up on local. If this is your first time using Cypress you will need to run the install script in your command line.
+
+### Command line for download
+```
+npm install cypress --save-dev
+```
```
Comment (rayestrada) [inline testing/docs/gettingStarted.md:12]: For the intialization in your local I believe you just need to run `npm install` cypress is already a listed dependency and will be installed along with the other dependencies.

File: testing/docs/gettingStarted.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+| `npm run cy:axe` | Run accessability testing with Cypress Axe Plugin|
+| `npm run cy:links`   | Run link validator testing for internal links |
+| `npm run cy:proofer`   | Run link validator testing for external links |
+| `npm run cy:createUser`   | ^^ Run Create User command for Drupal Backend |
+| `npm run cy:deleteUser`   | ^^^ Run Delete User command for Drupal Backend |
+| `npm run testSuite`   | Run full testing suite|
+
+
+Note: 
+- ^ this will run all function tests headless and have the `createUser` and `delteUser` command already added.
```
Comment (rayestrada) [inline testing/docs/gettingStarted.md:32]: typo `deleteUser`

**Review by rayestrada — CHANGES_REQUESTED** [review]

File: testing/docs/gettingStarted.md
Comment (clmedders) [reply testing/docs/gettingStarted.md:1]: moved and updated

File: testing/docs/gettingStarted.md (hunk 1 above)
Comment (clmedders) [reply testing/docs/gettingStarted.md:12]: updated to just list `npm install`

File: testing/docs/gettingStarted.md (hunk 2 above)
Comment (clmedders) [reply testing/docs/gettingStarted.md:32]: fixed typo

**Pull request merged**